TerriaJS/terriajs
# Clipping box

### What this PR does

- Implements clipping box.
- Adds `group` and `checkbox-group` dimension selectors.

TODOs:
- [ ] Workbench clipping UI changes requested by client
- [ ] Sync with latest `master`
  
### Test me
  
1. Open this [CI link](http://ci.terria.io/clipping-box/#clean&https://gist.githubusercontent.com/na9da/3031b76246f3d5f6802b3ec1d56afd05/raw/f43383f9bb57f77d15ec082520fa122aa2f02b04/clipping-box-tilesets.json).
2. Add any of the 2 3dtiles datasets.
3. Zoom to the dataset and enable the clipping box for the item from the workbench
4. Drag, scale and rotate the clipping box to select the clipping volume
5. Create a share link and open and verify that the clipping is retained.

### Checklist
